features/breol_search_frontend/breol_search_frontend.pages_default.inc

```diff
@@ -29,6 +29,14 @@ function breol_search_frontend_default_page_manager_handlers() {
     'contexts' => array(),
     'relationships' => array(),
     'name' => '',
+    'metatag_panels' => array(
+      'enabled' => 1,
+      'metatags' => array(
+        'title' => array(
+          'value' => 'Søgning | %keywords:html_safe | [current-page:pager][site:name]',
```

Comment:
You should use the the t() function and keep the english strings

Reply:
Of course! I've fixed this and then we have to fix our translation which are off.